setup/setup.md

```diff
@@ -0,0 +1,41 @@
+# Video Setup for MtlPy
```

Comment:
What about renaming this from `setup/video.md` to `setup/video.md` ? 

Or even `docs/video_setup.md` ?